.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/ambv/black
```

Comment:
Can you update this: https://github.com/psf/black

Reply:
Also, can you run all your new files through black once you're finished with the rest of the comments.

Reply:
No problem! I will get started on finishing up the wflow test.